3.10/http/general.md

```diff
@@ -611,3 +611,41 @@ receiving Coordinator will also check the header on its own.
 Apart from that, the header will not be included in cluster-internal requests
 executed by the Coordinator, e.g. when the Coordinator issues sub-requests
 to DB-Servers or Agency instances.
+
+Responding to liveliness probes
+-------------------------------
+
+You can now optionally start the HTTP REST interface of arangod instances early or late 
+during the startup process. ArangoDB versions before 3.10 only allowed starting the interface
+late, after the recovery was finished.
```

Comment:
It's not necessarily clear to users what the recovery is and that it's always run, not just after a crash.